packages/nx/src/utils/package-manager.ts

```diff
@@ -242,10 +242,11 @@ export function getPackageManagerCommand(
  */
 export function getPackageManagerVersion(
   packageManager: PackageManager = detectPackageManager(),
-  cwd = process.cwd()
+  cwd = process.cwd(),
+  forceExecPackageManager = false
```

Comment:
Let's not add this and keep things simple, let's just run execSync(`npm --version`) in the implementation where we need it. It doesn't need a generic solution